docassemble/AssemblyLine/data/questions/al_terms_of_use.yml

```diff
@@ -0,0 +1,72 @@
+---
+code: |
+  # Customize this with the preferred contact information. It could be a physical address
+  # rather than an email.
+  AL_DEFAULT_CONTACT_INFORMATION = get_config("server administrator email", AL_ORGANIZATION_TITLE)
+---
+code: |
+  AL_PRODUCTION_SITE_HOSTING_PROVIDER = AL_ORGANIZATION_TITLE
```

Comment:
nit: I think `AL_HOSTING_PROVIDER` is succinct enough; I think `PRODUCTION_SITE` is implied / is a lil misleading (the hosting provider is also hosting the dev site likely, right? The terms could be dynamic to the server).